[2022-03-17 03:19:12,548] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-01-02T06:00:00+00:00 [queued]>
[2022-03-17 03:19:12,586] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-01-02T06:00:00+00:00 [queued]>
[2022-03-17 03:19:12,588] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:19:12,589] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:19:12,591] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:19:12,617] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-01-02T06:00:00+00:00
[2022-03-17 03:19:12,623] {standard_task_runner.py:52} INFO - Started process 796 to run task
[2022-03-17 03:19:12,632] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-01-02T06:00:00+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmpgl9f_jop', '--error-file', '/tmp/tmpdlwjwc3h']
[2022-03-17 03:19:12,639] {standard_task_runner.py:77} INFO - Job 14: Subtask ingest
[2022-03-17 03:19:12,746] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-01-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:19:12,857] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-03-17 03:19:12,860] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:19:12,864] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:19:12,877] {bash.py:169} INFO - Output:
[2022-03-17 03:19:12,947] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:19:12,948] {bash.py:173} INFO - dags
[2022-03-17 03:19:12,950] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:19:12,952] {bash.py:173} INFO - logs
[2022-03-17 03:19:12,954] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:19:12,958] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:19:12,959] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:19:12,961] {bash.py:173} INFO - output_2019-04.csv
[2022-03-17 03:19:12,962] {bash.py:173} INFO - output_2019-05.csv
[2022-03-17 03:19:12,966] {bash.py:173} INFO - plugins
[2022-03-17 03:19:12,970] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:19:12,972] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:19:12,973] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:19:13,027] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190102T060000, start_date=20220317T031912, end_date=20220317T031913
[2022-03-17 03:19:13,083] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:19:13,126] {local_task_job.py:146} INFO - Task exited with return code 0
